Support output message for the `call_files_to_save` action (#2599)
kyma-project/cli: docs/extensibility/actions.md

| Name | Description |
| --- | --- |

> | **outputMessage** | Message that will be printed to the terminal after saving all files |

Reviewer: ```suggestion
| **outputMessage** | Message that is printed to the terminal after saving all files |
```
